swig/README.md

```diff
@@ -162,27 +162,26 @@ for (int i = 0; i < NUM_TIMES; i++) {
 }
 ```
 
-This works because here `cg` gets destructed each time it goes out of scope. The same is not true in Scala
+This works because here `cg` gets destructed each time it goes out of scope. The same is not true in Scala. If you were to try something like
 
 ```scala
-// BAD! DO NOT DO THIS!
+// THIS CODE WILL NOT RUN
```

Comment:
Maybe just delete the paragraph on `new ComputationGraph` now since it's irrelevant?